packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -214,18 +214,26 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
       privateArgs.assetInfo,
     );
   }
-
+  const nobleAccountV = zone.makeOnce('NobleAccount', () => makeNobleAccount());
   const feedKit = zone.makeOnce('Feed Kit', () => makeFeedKit());
 
   const poolAccountV = zone.makeOnce('PoolAccount', () => makeLocalAccount());
   const settleAccountV = zone.makeOnce('SettleAccount', () =>
     makeLocalAccount(),
   );
   // when() is OK here since this clearly resolves promptly.
-  /** @type {HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>[]} */
-  const [poolAccount, settlementAccount] = await vowTools.when(
-    vowTools.all([poolAccountV, settleAccountV]),
+  /** @type {[HostInterface<OrchestrationAccount<{chainId: 'noble-1';}>>, HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>, HostInterface<OrchestrationAccount<{chainId: 'agoric';}>>]} */
```

Comment:
chainId is `noble-1` but not `agoric-3`?

Reply:
This is some tech debt that would hopefully be solved in #9992. `agoric` is a special case in the types currently since checks do ~startsWith (to support `agoriclocal` and `agoric-3`).

Agree this is confusing, updated to use `agoric-3` here.